docs/resources/gateway_plugin_ai_mcp_proxy.md

```diff
@@ -60,8 +60,10 @@ resource "konnect_gateway_plugin_ai_mcp_proxy" "my_gatewaypluginaimcpproxy" {
             # ...
           ]
         }
-        request_body = "...my_request_body..."
-        scheme       = "https"
+        request_body = {
+          key = jsonencode("value")
+        }
```

Comment:
This feels strange as a consumer. Can we either define a schema for `additionalProperties`, or mark the entire thing as `any` so that the config is:

```suggestion
        request_body = jsonencode({
          key = "value"
        })
```

Using `jsonencode` on each value feels wrong